### ISSUE TITLE
Scoping

### ISSUE THREAD

**Issue by tuurma** [LingSIG/wordAttributes#1]
Dear Piotr, Susanne and Martin (who brought this thingy to my attention, thanks),

I think a good exercise before discussing the actual body of the proposal and its ultimate phrasing would be to decide what the audience and goal is. The experience of TEI Simple is people have one zillion troubles with it, many stemming from their fear of 'one size, doesn't fit anybody' approach.

So, my questions would be:

1. Do you want this to become exemplar customization or is it meant to be incorporated into tei_all schema?
2. Who is it for?

> "for those who require only these simple means"

was the vague answer for 2 provided in the Wiki. It is very similar to what TEI Simple was stating in its aims and while I, personally, see the virtue in it, I also heard from the Council and elsewhere that it is not helpful for those who encounter such a customization to determine if it is any good for their purposes.

**Comment by bansp** [comment]
Hi Magda,
Thanks for your input!
As for the audience, the fragment that you yanked from the wiki is pretty well contextualized there, I think :-) 
As for your item 1, thanks, it's a valid issue to which I personally don't have a clear answer yet. The whole idea has a flavour of customization for a particular group of users (but then, so is genetic encoding, which is in tei_all all the way). On the customization scenario, I would be afraid of / unhappy with two issues, minimally: (1) that having to prefix attribute names with some extra namespace would defeat the (part of the) original purpose of attracting corpus linguists who are afraid of the extremely robust TEI machinery and (2) I would love to see functional symmetry (or at least very clear relations) between the members of (i) this proposed group of attributes, (ii) the gramGrp elements of dictionary entries and (iii) the potentially extended grammatical description of standoff approaches. 
Bottom line: I think I would like to make these attributes as native to the TEI as it gets. In other words, personally, I would target tei_all.

**Comment by bansp** [comment]
Thanks again for your input, closing the issue after the [merger with TEI/dev](https://github.com/TEIC/TEI/commit/b7bb4e5f8de534a3b3cd1b73501fcf57e6264ad0).
